src/Symfony/Component/Validator/Tests/Constraints/ChoiceValidatorTest.php

```diff
@@ -112,8 +112,8 @@ public function provideConstraintsWithCallbackFunction(): iterable
         ])];
         yield 'doctrine style, static method' => [new Choice(['callback' => [__CLASS__, 'staticCallback']])];
         yield 'named arguments, namespaced function' => [new Choice(callback: 'Symfony\Component\Validator\Tests\Constraints\choice_callback')];
-        yield 'named arguments, closure' => [new Choice(callback: fn () => ["foo", "bar"])];
-        yield 'named arguments, static method' => [new Choice(callback: ["Symfony\Component\Validator\Tests\Constraints\ChoiceValidatorTest", "staticCallback"])];
+        yield 'named arguments, closure' => [new Choice(callback: fn () => ['foo', 'bar'])];
+        yield 'named arguments, static method' => [new Choice(callback: ["Symfony\Component\Validator\Tests\Constraints\ChoiceValidatorTest", 'staticCallback'])];
```

Comment:
```suggestion
        yield 'named arguments, static method' => [new Choice(callback: ['Symfony\Component\Validator\Tests\Constraints\ChoiceValidatorTest', 'staticCallback'])];
```
?

Reply:
Apparently, CS Fixer did not pick that up. 🤔